draft-ietf-acme-acme.md

```diff
@@ -355,14 +355,14 @@ network terminate TLS connections from clients so that it can inspect client
 requests for denial-of-service protection.
 
 These intermediaries can also change values in the request that are not signed
-in the HTTPS request, e.g., the request URI and headers.  ACME uses JWS to
+in the HTTPS request, e.g., the request URL and headers.  ACME uses JWS to
 provide an integrity mechanism, which protects against an intermediary
-changing the request URI to another ACME URI.
+changing the request URL to another ACME URL.
```

Comment:
Instead of "to another ACME URL" what about "to the URL of another ACME resource"? I think that might help clarify this sentence.